gbn/docker-compose.yml

```diff
@@ -4,11 +4,13 @@ volumes:
   gbn-connector-logs:
   gbn-connector-db-data:
   gbn-store-db-data:
+  bridgehead-proxy:
+  blaze-data:
 
 services:
   traefik:
     container_name: bridgehead-traefik
-    image: traefik:2
+    image: traefik:latest
```

Comment:
I would not recommend latest. Last time traefik had a major release they changed the configuration syntax which broke old deployments.
An automatic update for this would result in all bridgeheads going down.

Reply:
Please use latest (see below)